app/models/setting/content.rb

```diff
@@ -4,7 +4,7 @@ class Setting::Content < Setting
   def self.load_defaults
     return unless super
 
-    BLANK_ATTRS.concat %w(register_hostname_fact default_location_subscribed_hosts
+    BLANK_ATTRS.concat %w(register_hostname_fact default_location_subscribed_hosts default_location_puppet_content
```

Comment:
default_location_puppet_content is being listed twice.

Reply:
strange....